### PR TITLE
feat: add brewfile to install required tools

### PR DESCRIPTION
**Pull request by markusfassbender**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues
 
As developer one needs to install a bunch of tools via brew, so that Xcode building can work.

### Solutions

The Brewfile helps to do all at once and have a list of the required tools.

#### How to Test

Run `brew bundle` on your machine.

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by johnxnguyen — APPROVED** [review]
Wonderful!

File: Brewfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+brew "carthage"
+brew "sourcery"
+brew "swiftgen"
+brew "swiftlint"
```
Comment (caldrian) [inline Brewfile]: SwiftLint shouldn't be required anymore, since PR https://github.com/wireapp/wire-ios/pull/653

Comment (markusfassbender) [reply Brewfile]: Thanks!

**Review by agisilaos — APPROVED** [review]
Looks good to me! :)

**Pull request merged**